charts/ks-devops/charts/jenkins/templates/jenkins-casc-config.yml

```diff
@@ -201,7 +201,11 @@ data:
           size: 20
           ttl: 300
         enabled: {{ .Values.ksAuth.enabled }}
-        server: "http://{{ .Release.Name }}-ks-devops-apiserver.{{ .Release.Namespace }}:9090/"
+        {{- if .Values.ksAuth.server }}
+        server: {{ .Values.ksAuth.server }}
+        {{ else }}
+        server: "http://{{ .Release.Name }}-apiserver.{{ .Release.Namespace }}:9090/"
```

Comment:
There still a name conflict of apiserver. Please see [here](https://github.com/kubesphere/ks-devops/blob/6102c5eb1b7b1515266bc6731c86e1a83179720a/charts/ks-devops/templates/_helpers.tpl#L8-L24).

Reply:
I suggest we create a service with a specific name for Jenkins to use